### ISSUE TITLE
Audio output

### ISSUE THREAD

**Issue by HerbstGewinn** [mdagost/openai-realtime-streamlit#4]
Hi, currently the app can receive audio input and text input (in json format), right ? 

I just wanted to know, if it is also possible to have the conversations, including audio output from the realtime api ? 

Or maybe if I am doing something wrong, or if it is not included in this POC ? 

Like th eproject though extremely and its cool to integrate it in streamlit !

**Comment by mdagost** [comment]
Yep--the streaming audio should be playing.  Do you not hear it?
